hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/ConsumerTestRuntimeEnvironment.java

```diff
@@ -285,7 +284,7 @@ private WorkloadSupervisor startNode(WorkloadSupervisor workloadSupervisor) {
     }
 
     void waitForRegistration(String consumerId) {
-        await().atMost(adjust(ONE_SECOND)).until(() -> isRegistered(consumerId));
+        await().atMost(adjust(Duration.ofSeconds(1))).until(() -> isRegistered(consumerId));
```

Comment:
I wondered if we want to define some common constants for ONE_SECOND, FIVE_..., TEN, etc. It is used across all tests. On the other hand it is not something which will change for all tests at the same time.